src/client/eppo-client.ts

```diff
@@ -165,8 +160,25 @@ export default class EppoClient implements IEppoClient {
         flagKey,
         subjectAttributes,
         assignmentHooks,
-        ValueType.StringType,
-      )?.stringValue ?? null
+        ValueType.JSONType,
+      ).stringValue ?? null
+    );
+  }
+
+  public getParsedJSONAssignment(
+    subjectKey: string,
+    flagKey: string,
+    subjectAttributes: Record<string, EppoValue> = {},
+    assignmentHooks?: IAssignmentHooks | undefined,
+  ): object | null {
+    return (
+      this.getAssignmentVariation(
+        subjectKey,
+        flagKey,
+        subjectAttributes,
+        assignmentHooks,
+        ValueType.JSONType,
+      ).objectValue ?? null
```

Comment:
So we pull `stringValue` in `getJSONStringAssignment` and `objectValue` in `getParsedJSONAssignment`